### PR TITLE
Some extra tests around multiple-valued link handling in navigation

### PR DESCRIPTION
**Pull request by mjc**
In the process of trying to track down a bug that turned out not to be in ExHal, I wrote these extra tests that might be useful.

### CODE REVIEW

File: test/exhal/transcoder_test.exs [hunk 1]
```diff
@@ -145,3 +145,91 @@ defmodule ExHal.TranscoderTest do
...
+   end
+
+   assert %{} == MyOtherMultiLinkTranscoder.decode!(doc)
+
+   encoded = MyOtherMultiLinkTranscoder.encode!(%{tag: []})
+   refute ExHal.Document.has_link?(encoded, "tag")
+  end
+end
+
+defmodule ExHal.TranscoderAbsentLinksTest do
```
Comment (pezra) [inline test/exhal/transcoder_test.exs]: Is this a duplicated of [line 149](https://github.com/pezra/exhal/pull/36/files#diff-9defe435edba51bde3e9f11bee304c3dR149)?

Comment (mjc) [inline test/exhal/transcoder_test.exs]: Whoops. :)

Comment (pezra) [inline test/exhal/transcoder_test.exs]: OIC, mind just deleting this line: https://github.com/pezra/exhal/pull/36/files#diff-9defe435edba51bde3e9f11bee304c3dR216

**Comment by pezra** [comment]
Closing this as it doesn't seem to be changing. Please reopen if you want to make the changes discussed.

**Pull request closed without merging**